.travis.yml

```diff
@@ -1,10 +1,13 @@
+dist: xenial
+os: linux
 cache: bundler
-sudo: false
 language: ruby
 rvm:
-  - 2.3.5
-  - 2.4.0
-  - 3.0.0
+  - 2.6.0
+  - 2.6.5
+  - 2.6.10
```

Comment:
Is there any reason to test with several patch versions of 2.6, i think with just the latest should be enough for the case.

Reply:
Nope... I was just trying to cover what I consider bare minimum for now, plus one that is common in our projects as well as the last point release in 2.6. I also now I am going to follow up with another PR soon that will change this all again and figure I could clean this up then. Mostly this was just to prove my current changes work back that far. 
I plan to update and clean up this list in the next PR where we start to lock down a minimum supported version.

Short answer just being overly cautious and enjoyed watching all the test runs...